### DISCUSSION TITLE
Using aqua with Devcontainers emerging features standard might provide a great experience

### DISCUSSION THREAD

**Discussion by sheldonhull** [aquaproj/aqua#1273] [Deprecated (Feature Request)] [answered]
https://containers.dev/features

The process is more bash driven, however, having a feature that is able to call aqua project, setup the required env variables, and provide json parameter input of the desired packages to bootstrap into the config might be a promising option. Lots of ways to slice this, but wanted to pin this as an idea in case I get time or someone else does to look into this.

**Comment by suzuki-shunsuke** [comment]
Thank you for your proposal.
I'm not familiar with Devcontainers, so I'll take a look when I have a time.

**Comment by suzuki-shunsuke** [comment]
Moved this discussion to https://github.com/aquaproj/aqua/issues/1659

Background: https://github.com/aquaproj/aqua/issues/1666

**Comment by suzuki-shunsuke** [comment, marked as answer]
I've released the Dev Container Feature aqua-installer 🎉

- https://github.com/aquaproj/devcontainer-features
- https://github.com/aquaproj/aqua-renovate-config/releases/tag/2.3.0
  - https://github.com/aquaproj/aqua-renovate-config/pull/563
  - https://github.com/aquaproj/aqua-renovate-config/pull/564
- https://github.com/aquaproj/aquaproj.github.io/pull/945

Please read [the document](https://github.com/aquaproj/devcontainer-features/tree/main/src/aqua-installer).

**Reply by kevcube** [reply]
Hi @suzuki-shunsuke, thanks for creating the devcontainer feature!

Could you please PR to add it to the official list? 

https://github.com/devcontainers/devcontainers.github.io/blob/gh-pages/_data/collection-index.yml
https://containers.dev/features

**Reply by suzuki-shunsuke** [reply]
Sure. I've created a pull request.

- https://github.com/devcontainers/devcontainers.github.io/pull/482
